core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -276,7 +277,7 @@
         </div>
       </div>
       <div [hidden]="!learnSubmenuIsShown" class="oppia-community-library-section">
-        <a href="/community-library" [smartRouterLink]="'/' + PAGES_REGISTERED_WITH_FRONTEND.LIBRARY_INDEX.ROUTE" class="e2e-mobile-test-library-link">
+        <a href="/community-library" [smartRouterLink]="'/' + PAGES_REGISTERED_WITH_FRONTEND.LIBRARY_INDEX.ROUTE" tabindex="-1" class="e2e-mobile-test-library-link">
```

Comment:
Why is tabindex of -1 being added here? Why not 0?

Reply:
@Rijuta-s actually the button was being selected twice when tab was pressed, so `tabindex=-1` is used to prevent the 2nd selction using tab key.

Reply:
Thanks.